crates/hyperdrive-math/src/short/open.rs

```diff
@@ -864,26 +864,31 @@ mod tests {
             // Run the preamble.
             initialize_pool_with_random_state(&mut rng, &mut alice, &mut bob, &mut celine).await?;
 
+            // Set the variable rate to 0.
+            // This is required so that no interest is accrued between the
+            // estimate and actual open_short call.
+            alice.advance_time(fixed!(0), fixed!(1)).await?;
```

Comment:
this doesn't make sense, something else must be wrong.  this just makes sure that the vault share price isn't 1.  the short is estimated/opened after this so they should still be the same no?

Reply:
👍 added a comment per our discussion